src/app/components/NavigationBar/NavigationBar.tsx

```diff
@@ -181,7 +182,7 @@ export const NavigationBar = ({ title, profile, variant, menu, userActions }: Na
 			?.wallets()
 			.values()
 			.filter((wallet) => wallet.network().isLive());
-	}, [profile]);
+	}, [profile, profileWalletsCount]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Why is eslint disabled for this line?

Reply:
@faustbrian same as the above. Needed to react  navbar's state (disabled send/receive buttons etc) when profile wallets change, but it's not used directly inside the hook, and getting build errors on exhaustive-deps.